<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5L5Q2EV" failures="1" tests="2" name="Test2" time="218.709" errors="0" timestamp="2025-08-27T21:50:31 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verifyAccountRegistration" time="3.459">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC001_AccountRegistrationTest.verifyAccountRegistration(TC001_AccountRegistrationTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- verifyAccountRegistration -->
  <testcase classname="testCases.Tc002_LoginTest" name="verifyLogin" time="0.0">
    <skipped/>
  </testcase> <!-- verifyLogin -->
</testsuite> <!-- Test2 -->
